Quadriga/src/main/webapp/WEB-INF/views/top-navs/loggedin-nav.jsp

```diff
@@ -6,6 +6,7 @@
 <nav id="nav" class="skel-ui-fixed">
 	<ul>
 		<sec:authorize access="hasAnyRole('ROLE_QUADRIGA_USER_ADMIN', 'ROLE_QUADRIGA_USER_STANDARD', 'ROLE_QUADRIGA_USER_RESTRICTED', 'ROLE_QUADRIGA_USER_COLLABORATOR')"><li ${currentPage == "home" ? "class=\"current_page_item\"" : ""}><a href="${pageContext.servletContext.contextPath}/auth/home">Home</a></li></sec:authorize>
+		<sec:authorize access="permitAll"><li ${currentPage == "home" ? "class=\"current_page_item\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites">Sites</a></li></sec:authorize>
```

Comment:
this tab is active whenever some is on the 'home' page:
![image](https://cloud.githubusercontent.com/assets/8881141/14292798/7f7ef7e0-fb1e-11e5-98a1-38f27001abc5.png)
Also, move that tab to the end after "Networks"